Library/Homebrew/keg_relocate.rb

```diff
@@ -177,41 +177,34 @@ def recursive_fgrep_args
 
   def egrep_args
     grep_bin = "grep"
-    grep_args = recursive_fgrep_args
-    grep_args += "Pa"
+    grep_args = "-Pla"
```

Comment:
Are there long flags you can use here so it's a little more readable?

Reply:
I've switched the flags to now use the long versions.